ivy/container/manipulation.py

```diff
@@ -451,6 +494,49 @@ def flip(
         ivy.Container instance method variant of ivy.flip. This method simply wraps the
         function, and so the docstring for ivy.flip also applies to this method
         with minimal changes.
+
+        Parameters
+        ----------
+        self
+            input container.
+        axis
+            axis (or axes) along which to flip. If axis is None,
+            all input array axes are flipped. If axis is negative,
+            axis is counted from the last dimension. If provided
+            more than one axis, only the specified axes. Default: None.
+        key_chains
+            The key-chains to apply or not apply the method to. Default is None.
+        to_apply
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is True.
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is False.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples). Default is False.
+        out
+            optional output container, for writing the result to.
+            It must have a shape that the inputs broadcast to.
+
+        Returns
+        -------
+        ret
+            an output container having the same data type and
+            shape as ``self`` and whose elements, relative to ``self``, are reordered.
+
+        Examples
+        --------
+        With one :class:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=ivy.array([-1, 0, 1]),
+        ...                   b=ivy.array([2, 3, 4]))
+        >>> y = ivy.flip(x)
```

Comment:
As its a container instance method example, the instance call should be made i.e., `x.flip()`